src/activities/Elsa.Activities.Email/Elsa.Activities.Email.csproj

```diff
@@ -1,22 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
+    <Import Project="..\..\..\common.props" />
+    
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <PackageVersion>1.0.0</PackageVersion>
-        <LangVersion>latest</LangVersion>
-        <Authors>Elsa Contributors</Authors>
+        <TargetFramework>netstandard2.0</TargetFramework>       
         <Description>
             Elsa is a set of workflow libraries and tools that enable lean and mean workflowing capabilities in any .NET Core application.
             This package provides the following Email activities:
 
             * SendEmail
-        </Description>
-        <Copyright>2020</Copyright>
-        <PackageProjectUrl>https://github.com/elsa-workflows/elsa-core</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/elsa-workflows/elsa-core</RepositoryUrl>
-        <RepositoryType>GitHub</RepositoryType>
+        </Description>   
         <PackageTags>elsa, workflows, email, smtp</PackageTags>
         <PackageIcon>icon.png</PackageIcon>
+        <Nullable>disable</Nullable>

```

Comment:
Why do we disable Nullable for this project?

Reply:
Because it wasn't in there before. By default it is off.

Reply:
I see. As far as I’m concerned, we should enable nullable everywhere by default, so feel free to not bother with disabling it.

Reply:
I'm with you on that. But I would ignore the warnings for now, because I have another pull request which concerns SendMail. I want to avoid conflicts.